lib/try.js

```diff
@@ -1,17 +1,75 @@
 var express = require('express');
 var request = require('request');
+var streamBuffers = require("stream-buffers");
 
 module.exports = function() {
   var proxy = new express.Router();
   
-  proxy.all('/', function(req, res) {
+  proxy.all('/', function( req, resp ) {
+    var requestedUrl = getRequestedUrl( req );
+    enrichResponseWithRequestHeaders( req, resp );
+    var requestBuffer = new streamBuffers.WritableStreamBuffer({
```

Comment:
bodyParser generally can modify the request,

so instead of parsing and then serializing, it may be better to forward the request as it is